src/examples/dropdowns/MultiselectDefault.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useEffect, useRef } from 'react';
+import debounce from 'lodash.debounce';
+import {
+  Dropdown,
+  Multiselect,
+  Field,
+  Menu,
+  Item,
+  Hint,
+  Label
+} from '@zendeskgarden/react-dropdowns';
+import { Tag } from '@zendeskgarden/react-tags';
+
+const options = [
```

Comment:
@rossmoody I avoided adding Garden Ipsum as to not avoid conflicts with what you had in mind. Could you send me assets that reflect how these examples should look (with Garden Ipsum)?

Reply:
Working on this, one thing of note, it looks like the Debounce dependency is messing with the CodeSandbox